techniques/css/C30.html

```diff
@@ -216,26 +212,26 @@ <h2>Related Resources</h2>
 	
 
 <section id="related">
-			<h2>Related Techniques</h2>
+			<h2>関連テクニック</h2>
 			<ul>
 				<li><a href="../css/C29">C29: Using a style switcher to provide a conforming alternate version</a></li>
 				<li><a href="../failures/F3">F3: Failure of Success Criterion 1.1.1 due to using CSS to include images that convey
                     important information</a></li>
 			</ul>
 		</section>
 <section id="tests">
-			<h2>Tests</h2>
+			<h2>検証</h2>
       <section class="procedure" id="procedure">
-				<h3>Procedure</h3>
+				<h3>手順</h3>
 				<ol>
-					<li>Check that the web page includes a control that allows users to select an alternate presentation.</li>
-					<li>Check that when the control is activated the resulting page includes text (programmatically determined text) wherever images of text had been used.</li>
+					<li>ウェブページに利用者が代替となる提示を選択できるコントロールが含まれていることを確認する。</li>
+					<li>コントロールがアクティブである時、結果のページは文字画像が用いられている箇所全てにおいてテキスト (プログラムによる解釈が可能なテキスト) を含んでいることを確認する。</li>
```

Comment:
check that ... の訳を、他のテクニックと合わせて「チェックする」としました。
```suggestion
					<li>コントロールがアクティブである時、結果のページは文字画像が用いられている箇所全てにおいてテキスト (プログラムによる解釈が可能なテキスト) を含んでいることをチェックする。</li>
```